Add files via upload

altuhov-as opened the pull request.

altuhov-as committed:
Add files via upload

DmitryChitalov commented on task_1.py:
```diff
@@ -0,0 +1,64 @@
+        my_calc()
+
+
+my_calc()
```
выполнено

DmitryChitalov commented on task_2.py:
```diff
@@ -0,0 +1,32 @@
+    # вызываем функцию рекурсивно и суммируем результат
+    return tuple(map(sum, zip(count_even_odd(n // 10), result)))
+
+print(count_even_odd(n))
```
выполнено

DmitryChitalov commented on task_3.py:
```diff
@@ -0,0 +1,42 @@
+
+reverse_num( int( input("Input integer number: ")) )
+
+print(result)
```
1) грубые ошибки стиля - табуляция вместо пробелов и другие ошибки стиля
2) global result - антипаттерн. и он здесь не нужен def reverse_num(n : int, zero=False, result=0):

DmitryChitalov commented on task_4.py:
```diff
@@ -0,0 +1,24 @@
+count = int( input("Введите количество элементов: "))
+
+print(f"Количество элементов - {count}, их сумма - {sum_row2(count)}")
+
```
1) ошибка запуска
Введите количество элементов: 3
Traceback (most recent call last):
  File "C:/Users/Администратор/Desktop/ТЕСТОВЫЙ.py", line 11, in <module>
    print(f"Количество элементов - {count}, их сумма - {sum_row2(count)}")
NameError: name 'sum_row2' is not defined
2) грубые ошибки стиля - табуляция вместо пробелов и другие ошибки стиля

altuhov-as replied:
print(f"Количество элементов - {count}, их сумма - {sum_row2(count)}") - ошибка была здесь, **sum_row2** - название функции не правильное

DmitryChitalov commented on task_5.py:
```diff
@@ -0,0 +1,46 @@
+			lines += f'{s} - {chr(s)} '
+			print_ascii(s, f, lines, n)
+
+
```
дальше код не смотрю
нужна серьезная доработка

altuhov-as committed:
Update task_4.py

sum_row - исправил правильно имя функции

altuhov-as committed:
Create task_5.py

изменил код, обновил табуляцию в github

altuhov-as committed:
Update task_6.py

print(f'Не угадано! Загаданное число {secret}') - добавил скобки
переименовал переменую

altuhov-as committed:
Update task_7.py

Изменил табуляцию в настройках github
